common/flatpak-utils.c

```diff
@@ -5792,3 +5799,57 @@ flatpak_utils_ascii_string_to_unsigned (const gchar  *str,
     *out_num = number;
   return TRUE;
 }
+
+void
+(flatpak_log_change) (const char *file,
+                      int line,
+                      const char *func,
+                      const char *change,
+                      int opid,
+                      const char *installation,
+                      const char *remote,
+                      const char *ref,
+                      const char *commit,
+                      const char *format,
+                      ...)
+{
+#ifdef HAVE_LIBSYSTEMD
+
+  char message[1024];
+  va_list args;
+
+  va_start (args, format);
+  g_vsnprintf (message, sizeof (message), format, args);
+  va_end (args);
+
+  if (opid)
+    sd_journal_send ("MESSAGE_ID=" FLATPAK_MESSAGE_ID,
+                     "PRIORITY=5",
+                     "FLATPAK_VERSION=" PACKAGE_VERSION,
+                     "OBJECT_PID=%d", opid,
```

Comment:
Does "OBJECT_PID" have a pre-defined meaning? Its a bit weird otherwise.

Reply:
It is defined by the journal, for 'acting on behalf of somebody else' - setting it makes journald pick up a bunch of other fields. See systemd.journal-fields(7)

Reply:
Ok, that seems useful.